Clean up toString implementations

gregw opened the pull request:
Clean up toString implementations to mostly use TypeUtil.toShortName

gregw committed:
Avoid dump cycles

Fix #13003 by avoiding dump cycles

gregw committed:
Avoid dump cycles

Fix #13003 by avoiding dump cycles

gregw committed:
Avoid dump cycles

Fix #13003 by avoiding dump cycles

gregw committed:
Avoid dump cycles

Fix #13003 by avoiding dump cycles
Make toString methods more uniformly use `toShortName`

gregw committed:
Avoid dump cycles

Fix #13003 by avoiding dump cycles
Make toString methods more uniformly use `toShortName`

gregw committed:
Avoid dump cycles

Fix #13003 by avoiding dump cycles
Make toString methods more uniformly use `toShortName`

gregw committed:
Avoid dump cycles

Fix #13003 by avoiding dump cycles
Make toString methods more uniformly use `toShortName`

gregw committed:
Avoid dump cycles

Fix #13003 by avoiding dump cycles
Make toString methods more uniformly use `toShortName`

sbordet commented:
@gregw this PR has 296 file changes.
Can you split it in 2: one where you do the cleanups, and one where you handle the dump cycles?

gregw committed:
Avoid dump cycles

Fix #13003 by avoiding dump cycles
Make toString methods more uniformly use `toShortName`

gregw committed:
Avoid dump cycles

Fix #13003 by avoiding dump cycles
Make toString methods more uniformly use `toShortName`
